### PR TITLE
Victor Lazov - formatting code

### PR DESCRIPTION
**Pull request by victorlazov**

### CODE REVIEW

File: formatting-code/v.lazov/JavaScript-code.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+b = navigator.appName;
+addScroll = false;
+
+if (navigator.userAgent.indexOf('MSIE 5') > 0 ||
```
Comment (asparuh-krastev) [inline formatting-code/v.lazov/JavaScript-code.js:4]: This has to do with addScroll, should be in the same paragraph with addScroll = false;

File: formatting-code/v.lazov/JavaScript-code.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+}
+
+var off = 0;
+var txt = "";
+var pX = 0;
+var pY = 0;
+
+document.onmousemove = mouseMove;
+
+if (b == "Netscape") {
```
Comment (asparuh-krastev) [inline formatting-code/v.lazov/JavaScript-code.js:16]: Same thing

File: formatting-code/v.lazov/JavaScript-code.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  if (b == "Netscape") {
+    pX = evn.pageX - 5;
+    pY = evn.pageY;
+  }
+  else {
+    pX = event.x - 5;
+    pY = event.y;
+  }
+
+  if (b == "Netscape") {
```
Comment (asparuh-krastev) [inline formatting-code/v.lazov/JavaScript-code.js:30]: I personally refactored this in only 1 if-else

File: formatting-code/v.lazov/Manager.php [hunk 4]
```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace League\CLImate\Settings;
+
+  class Manager {
```
Comment (asparuh-krastev) [inline formatting-code/v.lazov/Manager.php:5]: Why the indent?

File: formatting-code/v.lazov/JavaScript-code.js (hunk 3 above)
Comment (victorlazov) [inline formatting-code/v.lazov/JavaScript-code.js:30]: The homework emphasises on code formatting, not on logic.

File: formatting-code/v.lazov/Manager.php (hunk 4 above)
Comment (victorlazov) [inline formatting-code/v.lazov/Manager.php:5]: Class indent after a namespace.

File: formatting-code/v.lazov/JavaScript-code.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
+b = navigator.appName;
+addScroll = false;
+
+if (navigator.userAgent.indexOf('MSIE 5') > 0 ||
+    navigator.userAgent.indexOf('MSIE 6') > 0) {
```
Comment (akraychev) [inline formatting-code/v.lazov/JavaScript-code.js:5]: I think ) { should be on next line

File: formatting-code/v.lazov/JavaScript-code.js [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+var off = 0;
+var txt = "";
+var pX = 0;
+var pY = 0;
+
+document.onmousemove = mouseMove;
+
+if (b == "Netscape") {
+  document.captureEvents(Event.MOUSEMOVE)
```
Comment (akraychev) [inline formatting-code/v.lazov/JavaScript-code.js:17]: missing ;

File: formatting-code/v.lazov/Manager.php [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+      return FALSE;
+    }
+
+    public function settings() {
+      return [];
+    }
+
+    public function importSetting($setting) {
+      $short_name = basename(str_replace('\\', '/', get_class($setting)));
```
Comment (akraychev) [inline formatting-code/v.lazov/Manager.php:36]: I think it will be more readable if you separate on different lines

**Pull request merged**